src/components/Divider/Divider.tsx

```diff
@@ -0,0 +1,41 @@
+import { colors, borders } from '@magnetis/astro-tokens';
+import React from 'react';
+import { View } from 'react-native';
+
+import type { DividerOrientation, DividerType, DividerSize } from './types';
+
+export type DividerProps = {
+  orientation?: DividerOrientation;
+  type?: DividerType;
+  size: DividerSize;
+  testID?: string;
+};
+
+/**
+ * A customizable divider component.
+ *
+ * @param {DividerProps} props - The props for the Divider.
+ * @param {DividerOrientation} props.orientation - The orientation of the Divider.
+ * @param {DividerSize} props.size - The size of the Divider, based on magnetis/astro-tokens borders tokens keys.
+ * @param {DividerType} props.type - The type of the DIvider.
+ * @param {string} props.testID - Test ID for testing purposes.
+ *
+ * @returns {JSX.Element} The rendered Divider component.
+ * @example
+ * <Divider size='thick' />
+ */
+function Divider({ orientation = 'horizontal', type = 'neutral', size, testID = 'Divider' }: DividerProps) {
+  const baseProps = {
+    backgroundColor: type === 'neutral' ? colors.transparentFaintClear : colors.transparentBrightClear,
+    height: orientation === 'horizontal' ? borders[size] : '100%',
+    width: orientation === 'vertical' ? borders[size] : '100%',
+  };
+
+  return (
+    <View style={{ alignItems: 'center', justifyContent: 'center' }}>
+      <View style={{ ...baseProps }} testID={`${testID}.Line`}></View>
```

Comment:
⛏️ 
It's very important that `testID` prop must be used explicitly. In a test case, can be used in a `getByTestId` statement
```suggestion
      <View style={{ ...baseProps }} testID={testID}></View>
```

Reply:
Fixed in [61de2ac](https://github.com/magnetis/astro-native/pull/782/commits/61de2acb8a0d6e0340f134e916e88bc4e2aa5242)